core/modules/mysql/MySqlConfig.cc

```diff
@@ -28,118 +28,75 @@
 #include <fstream>
 #include <iostream>
 #include <sstream>
-#include <stdlib.h> // atoi
+#include <stdexcept>
 
+// LSST headers
+#include "lsst/log/Log.h"
+
+// Qserv headers
+#include "sql/SqlConnection.h"
+
+namespace {
+
+    LOG_LOGGER getLogger() {
+        static LOG_LOGGER logger = LOG_GET("lsst.qserv.mysql.MySqlConfig");
+        return logger;
+    }
```

Comment:
I believe we decided to use this construct everywhere:

    LOG_LOGGER _log = LOG_GET("lsst.qserv.mysql.MySqlConfig");